Products/ATContentTypes/setuphandlers.py

```diff
@@ -12,6 +12,12 @@
 from zope.i18n.locales import locales
 from plone.registry.interfaces import IRegistry
 
+try:
+    from Products.CMFPlone.interfaces import ILanguageSchema
```

Comment:
Would it not be better to use api.env.plone_version() here? I see that plone.api is not a dependancy of ATContentTypes, and I forget what conclusion the community reached on including plone.api in 'core' products. It may make things a little easier to read though.